lib/measures/CreateTypicalBuilding/tests/output/hvac_zone_mapping_2_psz_ac.osm

```diff
@@ -1,8318 +0,0 @@
-
-OS:Version,
```

Comment:
Are these models not needed anymore?

Reply:
They're outputs from running the test script but not important to the measure. [This](https://jira.pnnl.gov/jira/browse/OSSTD-477) ticket requested they be removed.

Reply:
Gotcha!